<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filterSubjectsDialog</class>
 <widget class="QDialog" name="filterSubjectsDialog">
  <property name="enabled">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>175</width>
    <height>32</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Filter Subjects</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res/resources.qrc">
    <normaloff>:/planner.png</normaloff>:/planner.png</iconset>
  </property>
  <layout class="QFormLayout" name="dialogFormLayout">
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="clearFilterBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Clear Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyFilterBtn">
       <property name="text">
        <string>Apply Filter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
